source/camera.c

```diff
@@ -164,9 +160,10 @@ Result http_get(char *url, char *path)
         ret = httpcOpenContext(&context, HTTPC_METHOD_GET, url, 1);
         ret = httpcSetSSLOpt(&context, SSLCOPT_DisableVerify); // should let us do https
         ret = httpcSetKeepAlive(&context, HTTPC_KEEPALIVE_ENABLED);
-        ret = httpcAddRequestHeaderField(&context, "User-Agent", "Anemone3DS/1.1.0");
+        ret = httpcAddRequestHeaderField(&context, "User-Agent", VERSION);
```

Comment:
Could this use Anemone3DS/Version instead of just the version string?

Reply:
```C
char* user_agent[0x64]; // I never know how much to allocate, but it shouldn't matter here
strcpy(user_agent, "Anemone3DS/ ");
strcat(user_agent, VERSION);
// [...]
free(user_agent);
```
Look good? This obviously goes above the loop, because allocating inside every iteration is `e x p e n s i v e`.

I'd push this myself, but a) I'm not confident with my C strings, and b) My credential helper is screwing with me.

Reply:
You should just use sprintf here

Reply:
Ah, yes, I forgot to remove the `*`.
```C
char user_agent[0x64];
sprintf(user_agent, "Anemone3DS/%s", VERSION);
```
I'd assume this would work?

Reply:
fixed with c7a8b23c100e7fc5611a5732facf3e616c42a08d, sorry. I derped and thought "Anemone3DS" already appeared in the version string on the bottom screen.